app/Config/core.php

```diff
@@ -246,7 +246,7 @@
  *
  * To use, prefix the CSS link URL with '/ccss/' instead of '/css/' or use HtmlHelper::css().
  */
-	//Configure::write('Asset.filter.css', 'css.php');
+    //Configure::write('Asset.filter.css', 'css.php');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Asset.filter.css', 'css.php');" but found "//Configure::write('Asset.filter.css', 'css.php');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment